omero/users/cli/import-bulk.rst

```diff
@@ -0,0 +1,99 @@
+Using bulk imports
+==================
+
+The CLI import option ``--bulk`` specifies a configuration file that
+can be used to perform a batch of imports with the same or similar
+options. The file is written in a simple YAML syntax and can be placed
+anywhere.
+
+A simple YAML file might look like: ::
+
+    ---
+    continue: "true"
```

Comment:
never really understood why this line is necessary. But it is again silently assumed it will be in every bulk.yml the users would create... Is this the same as the ``-k`` ? I would say no...

Should be explained and hinted that this is necessary for every bulk.yml if I am right.

Reply:
> Is this the same as the -k

Yes.

> Should be explained and hinted that this is necessary for every bulk.yml if I am right.

No. It is not needed.